### PR TITLE
chore: tests for cloud adapter base

### PR DESCRIPTION
**Pull request by joshgummersall**
Adds a set of unit tests for `CloudAdapterBase` functionality.

#minor

### CODE REVIEW

File: libraries/botbuilder-core/tests/cloudAdapterBase.test.js [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+        it('succeeds', function () {
+            new TestAdapter(BotFrameworkAuthenticationFactory.create());
+        });
+    });
+
+    describe('sendActivities', function () {
+        it('throws for bad args', function () {
+            assert.throws(() => adapter.sendActivities(undefined, []), { name: 'TypeError' });
+            assert.throws(() => adapter.sendActivities(new TurnContext(adapter), undefined), { name: 'TypeError' });
+            assert.throws(() => adapter.sendActivities(new TurnContext(adapter), []), { name: 'Error' });
```
Comment (mdrichardson) [inline libraries/botbuilder-core/tests/cloudAdapterBase.test.js:67]: nit: Should these `assert.throws()` also assert the correct error message? An edge case scenario where this might be valuable to check for would be a dev removing the argument check and the test throwing downstream for a separate reason, and then inappropriately passing.

Comment (joshgummersall) [reply libraries/botbuilder-core/tests/cloudAdapterBase.test.js]: Good call!

**Review by stevengum — APPROVED** [review]
